.circleci/config.yml

```diff
@@ -39,25 +39,38 @@ jobs:
       enabled: true
       image: circleci/classic:201710-01
       docker_layer_caching: true
-    parallelism: 3
     steps:
       - checkout
       - restore_cache:
           keys:
-          - v3-dependencies-{{ checksum "package.json" }}
+            - v3-dependencies-{{ checksum "package.json" }}
       - run: .circleci/install-yarn.sh
-      - run: 
+      - run:
           name: Get & run localnode image
           command: |
             LOCALNODE_SHA=8e36f81989cd0e17c9e6de035c4dde58f0f12bd5
             docker login -u $DOCKER_USER -p $DOCKER_PASS && docker pull oasisdex/localnode:$LOCALNODE_SHA
-            docker run -d -p 8545:8545 oasisdex/localnode:$LOCALNODE_SHA master
-      - run: 
+            docker run -d -p 8545:8545 oasisdexorg/localnode:$LOCALNODE_SHA master
+      - run:
           name: Wait for localnode to be up
           command: |
             ./.circleci/wait-for-it.sh localhost:8545 -t 10
             sleep 10
-      - run: 
+      - run:
```

Comment:
I have problems with my eyes or this is duplicate

Reply:
i know right! it's just a to stress test e2e setup. find tests that are failing etc

Reply:
so you want to have 2 setups running  :)  I feel you

Reply:
copying this few times was much faster than learning how to do proper loops in bash 😆